### ISSUE TITLE
Android: Default map after trying new Google Cloud-Based styling

### ISSUE THREAD

**Issue by octagonalsquare** [react-native-maps/react-native-maps#3732]
# Question

Ok, I can't find this issue any where, here or StackExchange. I am making an app and have 2 different styles depending on dark or light mode of the app. They worked great! But I was messing around with the style editor and decided to try the new cloud-based one to see if it had any new features or would make my life easier. I soon realized I couldn't properly integrate it with the react native maps and decided to just delete the style I made on the cloud and go back. But now the problem:
Since trying that, the android version of my app no longer uses the custom styles I made and am supplying it. It instead uses the default google map style. It doesn't do this on iOS. I assume it is because when I made the style in the cloud, it then was set to override the style when using my API key, and since I deleted the style in the cloud, my custom one in the app is being overridden by the default.

Any ideas how to fix this? Do I just need to make a new project with a new API key?

**Comment by octagonalsquare** [comment]
Update: Went ahead and made a new project/apikey and did all the SHA-1 linking and such (which is a pain for those who haven't done it) and it still uses the default. I don't have separate code for ios/android related to these styles so there should be no difference

**Comment by octagonalsquare** [comment]
Ok, so turns out the issue was simple, but still confusing. I had found that there is a property that isn't available on the standard Google Map Style maker called landscape.man_made.building. It is available in the cloud version. So I added it to my style as I noticed the buildings are some times hard to see due to them being the background color. It worked perfectly on iOS, but apparently it does not work on Android. Which seems backwards since Android only uses Google so you'd think it would be implemented as part of the Android Maps SDK before the iOS one.

Anyone know if this is an issue with the actual map SDK or if it is a react native maps limitation? If there is a work around, then I will use it as clarity and consistency between versions is important to this app
